torch_xla/csrc/helpers.cpp

```diff
@@ -21,6 +21,12 @@
 namespace torch_xla {
 namespace {
 
+static const bool experimental_unbounded_dynamism =
```

Comment:
Make this a function that returns the values of the flag, and call this function on places this variable is read.

Reason being, if it's a constant then it cannot be changed at runtime, and has to be set before the program starts.

Reply:
updated